test/spec/ol/interaction/dragzoom.test.js

```diff
@@ -32,7 +32,8 @@ describe('ol.interaction.DragZoom', function() {
       view: new View({
         projection: 'EPSG:4326',
         center: [0, 0],
-        resolution: 1
+        resolution: 1,
+        maxResolution: 8
```

Comment:
Why was `maxResolution: 8` added? Does the test pass without that addition too?

Reply:
This is not needed, will remove